tutorials/introduction/06-kolmogorov_equations.jmd

```diff
@@ -0,0 +1,118 @@
+---
+title:  Kolmogorov Backward Equations
+author: Ashutosh Bharambe
+---
+
+```julia
+using Flux, StochasticDiffEq
+using  NeuralNetDiffEq
+using Plots
+using Distributions
+```
+## Introduction on Kolmogorov Equations
+
+The one dimensional backward kolmogorov equation that we are going to deal with is of the form :
+$$
+\begin{equation}
+  \frac{\partial p}{\partial t} = \mu(x)\frac{\partial p}{\partial x} + \frac{1}{2}{\sigma^2}(x)\frac{\partial^2 p}{\partial x^2}
+\end{equation}
+$$
+Consider a Black Scholes Partial Differential Equation of the form :
+$$
+\begin{equation}
+  \frac{\partial V}{\partial t} + rS\frac{\partial V}{\partial S} + \frac{1}{2}{\sigma^2}{S^2}\frac{\partial^2 V}{\partial S^2} -rV = 0
```

Comment:
This is missing some important details. It's a terminal PDE, so it's defined with valuations on the end. How does this relate to the model? Describe the options market and where this idea comes from.